internal/engine/eval/jq/jq.go

```diff
@@ -88,9 +99,9 @@ func (jqe *Evaluator) Eval(ctx context.Context, pol map[string]any, res *engif.R
 		}
 
 		// Deep compare
-		if !reflect.DeepEqual(profileVal, dataVal) {
+		if !reflect.DeepEqual(numberToFloat64(expectedVal), numberToFloat64(dataVal)) {
```

Comment:
How does this work for booleans (like your example)?  I believe you that it does, but I was surprised to see that it worked.

Oh, I see!  `numberToFloat64` actually returns `any`, and passes through other values.  I'm trying to figure out if there's a better name -- maybe `standardizeNumbers`?

Reply:
Updated the name